### ISSUE TITLE
Doubled prefixed classnames

### ISSUE THREAD

**Issue by pandroid** [bradlc/tailwindcss-fluid#1]
First of all: I love your plugin... ;-) Thank you so much for sharing!
But there is a small issue. Maybe you can help.

I use a custom prefix in tailwinds config file
```  
options: {
  prefix: 'dhl-',
  important: false,
  separator: ':',
},
```

But with this your plugin generates classnames with doubled prefixes
```
.dhl-dhl-text-sm-fluid {
  font-size: 14px;
}

@media (min-width: 320px) {
  .dhl-dhl-text-sm-fluid {
    font-size: calc(12.22222px + 0.55556vw);
  }
}

@media (min-width: 1400px) {
  .dhl-dhl-text-sm-fluid {
    font-size: 20px;
  }
}
````
Am I doing something wrong?

**Comment by bradlc** [comment]
Hey @pandroid, thanks so much for the report!

This should now be fixed in 0.1.1

**Comment by pandroid** [comment]
Hey @bradlc, thanks a lot for the super fast fix. Now it works great.
